app/src/main/java/com/keyboardbuilder/dto/KeyboardCase.java

```diff
@@ -0,0 +1,39 @@
+package com.keyboardbuilder.dto;
```

Comment:
Given the design document, I created this class for one of the DTO's. 
1. Simple class for DTO.